scipy/interpolate/tests/test_interpolate.py

```diff
@@ -1012,7 +1012,7 @@ def test_read_only(self):
         p = PPoly(c, x, extrapolate='periodic')
 
         for writeable in (True, False):
-            xnew.flags.writeable = writeable
+            x.flags.writeable = writeable
```

Comment:
I'd make the both readonly.

Reply:
I set xnew to True and False, and x to False. Does that sound good?